src/rmp/src/blifParser.cpp

```diff
@@ -0,0 +1,120 @@
+#include "rmp/blifParser.h"
+
+#include <boost/bind.hpp>
+#include <boost/config/warning_disable.hpp>
+#include <boost/fusion/algorithm.hpp>
+#include <boost/fusion/container.hpp>
+#include <boost/fusion/include/adapt_struct.hpp>
+#include <boost/fusion/include/at_c.hpp>
+#include <boost/fusion/include/define_struct.hpp>
+#include <boost/fusion/include/io.hpp>
+#include <boost/fusion/sequence.hpp>
+#include <boost/fusion/sequence/intrinsic/at_c.hpp>
+#include <boost/lambda/lambda.hpp>
+#include <boost/optional/optional_io.hpp>
+#include <boost/spirit/include/phoenix_core.hpp>
+#include <boost/spirit/include/phoenix_operator.hpp>
+#include <boost/spirit/include/qi.hpp>
+#include <boost/spirit/include/qi_alternative.hpp>
+#include <iostream>
+#include <string>
+#include <vector>
+
+namespace blifParserClass {
```

Comment:
blif_parser would be more appropriate. no camel case for namespace, and they should not be so verbose